examples/hexagon_row.rs

```diff
@@ -59,23 +60,22 @@ fn startup(mut commands: Commands, asset_server: Res<AssetServer>) {
             storage: tile_storage,
             texture: TilemapTexture(texture_handle),
             tile_size,
-            mesh_type: TilemapMeshType::Hexagon(HexType::Row),
             ..Default::default()
         });
 }
 
-fn swap_mesh_type(mut query: Query<&mut TilemapMeshType>, keyboard_input: Res<Input<KeyCode>>) {
+fn swap_mesh_type(mut query: Query<&mut TilemapStorage>, keyboard_input: Res<Input<KeyCode>>) {
     if keyboard_input.just_pressed(KeyCode::Space) {
-        for mut tilemap_mesh_type in query.iter_mut() {
-            match *tilemap_mesh_type {
+        for mut tilemap_storage in query.iter_mut() {
```

Comment:
Does this even do anything anymore? From what I can tell the mesh type can't be changed.

Reply:
Yes, it does do something, because the mesh type can be changed because we provide a `set_mesh_type` method to change the mesh type. Then, when `extract` queries for `TilemapStorage`, it calls `get_mesh_type` on `TilemapStorage` which returns a different value.